yarn.lock

```diff
@@ -417,6 +417,14 @@ abbrev@1:
   resolved "https://registry.yarnpkg.com/abbrev/-/abbrev-1.1.1.tgz#f8f2c887ad10bf67f634f005b6987fed3179aac8"
```

Comment:
this package will depend on `core`, maybe better to start using `npm install` right away (so we don't have too much version changes when migrating from `yarn.lock` to `package-lock.json`

Reply:
will add a ticket for that